notes/release-train-issue-template.md

```diff
@@ -0,0 +1,72 @@
+Release Train Issue Template for Akka HTTP
+
+(Liberally copied and adopted from Scala itself https://github.com/scala/scala-dev/blob/b11cd2e4a4431de7867db6b39362bea8fa6650e7/notes/releases/template.md)
+
+For every Akka Http release, make a copy of this file named after the release, and expand the variables.
+Ideally replacing variables could become a script you can run on your local machine.
+
+Variables to be expanded in this template:
+- AKKA_HTTP_VER=??? (currently not used)
+
+Key links:
+  - akka/akka-http milestone: https://github.com/akka/akka-http/milestone/?
+
+### N weeks before the release
```

Comment:
Do we also want to start the train weeks before the release, or perhaps move a bit faster?

Reply:
Is it even known X weeks in advance?

Reply:
Hehe, good point. I just merged it with the "~ 1 week section"